scripts/test/deployAll/system-test-instance.properties

```diff
@@ -65,193 +65,23 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,GarbageCollectorStack,In
-# See Task CPU architecture at https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html
-sleeper.compaction.task.cpu.architecture=X86_64
-
-# The CPU and memory for a compaction task. Which of these are used will depend on the chosen CPU architecture.
-# See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html for valid options.
-sleeper.compaction.task.arm.cpu=1024
-sleeper.compaction.task.arm.memory=4096
-sleeper.compaction.task.x86.cpu=1024
-sleeper.compaction.task.x86.memory=4096
-
-# Flag to enable/disable storage of tracking information for compaction jobs and tasks.
-sleeper.compaction.status.store.enabled=true
-
-# The time to live in seconds for compaction job updates in the status store. Default is 1 week
-sleeper.compaction.job.status.ttl=604800
-
-# The time to live in seconds for compaction task updates in the status store. Default is 1 week
-sleeper.compaction.task.status.ttl=604800
-
-# The retention period for CloudWatch logs. See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html for valid options.
-# Use -1 to indicate infinite retention.
-sleeper.log.retention.days=30
-
-# The amount of memory in MB for the lambda that garbage collects old files
-sleeper.gc.memory=1024
-
-# The amount of memory in MB for the lambda that identifies partitions that need splitting
-sleeper.partition.splitting.finder.memory=2048
-
-# The timeout for the lambda that identifies partitions that need splitting in seconds
-sleeper.partition.splitting.finder.timeout.seconds=900
-
-# The amount of memory for the lambda that splits partitions
-sleeper.partition.splitting.memory=2048
-
-# The timeout for the lambda that splits partitions in seconds
-sleeper.partition.splitting.timeout.seconds=900
-
-# The amount of memory in MB for the lambda that executes queries
-sleeper.query.processor.memory=2048
-
-# The timeout for the lambda that executes queries in seconds
-sleeper.query.processor.timeout.seconds=900
-
-# The maximum number of simultaneous connections to S3 from a single client
-sleeper.s3.max-connections=25
-
-# The frequency with which the query processing lambda refreshes its knowledge of the system state
-# (i.e. the partitions and the mapping from partition to files), in seconds.
-sleeper.query.processor.state.refresh.period.seconds=60
-
-# The maximum number of records to include in a batch of query results send to
-# the results queue from the query processing lambda.
-sleeper.query.processor.results.batch.size=2000
-
-# Fargate version
-sleeper.fargate.version=1.4.0
-
-# The EC2 instance type to use for compaction tasks (when not using Fargate).
-sleeper.compaction.ec2.type=t3.xlarge
-
-# The minimum / initial desired / maximum number of instances for the cluster.
-sleeper.compaction.ec2.pool.minimum=0
-sleeper.compaction.ec2.pool.desired=0
 # Maximum number of EC2s limited for system test
 sleeper.compaction.ec2.pool.maximum=20
```

Comment:
Maybe this should be removed so that the default is used? It won't be deployed with EC2 anyway.

Reply:
Yes.